src/routes/console/supportWizard.svelte

```diff
@@ -85,7 +88,7 @@
             If you found a bug or have questions, please reach out to the Appwrite team. We try to
             respond to all messages within our office hours.
         </p>
-        <p class="text u-margin-block-start-32">Available: <b>Mon-Fri 09:00 - 17:00 UTC</b></p>
+        <p class="text u-margin-block-start-32">Available: <b>Mon-Fri, {supportTimings}</b></p>
```

Comment:
I can see cases where the day might change based on timezone too, right?

Reply:
You mean like the change in hours might change the working days, right?

Reply:
Done.
Example: Vladivostok Standard Time -

<img width="307" alt="VST" src="https://github.com/appwrite/console/assets/20625965/b97b6d42-ed58-4496-b50d-35adbdff0fc5">